python/lsst/sims/utils/CoordinateTransformations.py

```diff
@@ -319,11 +406,9 @@ def haversine(long1, lat1, long2, lat2):
 
     @param [out] the angular separation between points 1 and 2 in radians
 
-    From http://en.wikipedia.org/wiki/Haversine_formula
+    Ultimately just calls palpy.dsep
     """
-    t1 = np.sin(lat2 / 2.0 - lat1 / 2.0)**2
-    t2 = np.cos(lat1) * np.cos(lat2) * np.sin(long2 / 2.0 - long1 / 2.0)**2
-    return 2 * np.arcsin(np.sqrt(t1 + t2))
+    return _angularSeparation(long1, lat1, long2, lat2)
```

Comment:
Running on 10,000 coordinate pairs the old (pure numpy) method took 0.0018 seconds (total).  The new (palpy) method took 0.0078 seconds.  So, it's a factor of three slower, but it doesn't take a lot of time, regardless.  I made this change because the old method was giving NaNs near the poles and I figured palpy was mature enough to have fixed that.